lib/commands/setViewportSize.js

```diff
@@ -37,16 +36,19 @@ module.exports = function setViewportSize(size) {
         throw new ErrorHandler.CommandError('number or type of arguments don\'t agree with setViewportSize command');
     }
 
-    return setSize(this);
+    var shouldIndent = typeof(type) === 'undefined' ? true : type;
```

Comment:
As I can see in the doc [here](https://github.com/webdriverio/webdriverio/blob/master/lib/commands/setViewportSize.js#L20)
> param {Boolean}  type  set to `false` to change window size, `true` (default) to change viewport size

Looks like the default value of type is true. Should I edit this doc and change behaviour of type parameter?